src/layouts/TabLayout/TabLayout.js

```diff
@@ -76,6 +77,10 @@ const TabLayout = props => {
   const classes = useStyles();
   const { t } = useTranslation();
   const location = useLocation();
+  const languages = {
+    fr: 'Français',
```

Comment:
F id after E in alphabetical order, and to be consistent with Sign In view, I suggest to list in this same order.